lib/improver/nbhood_percentiles.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
```

Comment:
It looks as if a nbhood subdirectory has been added to improver to hold the nbhood code. This code will need to be moved in there